### ISSUE TITLE
Extension should offer to create a PR from a currently checked out branch.

### ISSUE THREAD

**Issue by dbaeumer** [Microsoft/vscode-pull-request-github#283]
Title says it all.

**Comment by natefinch** [comment]
Yeah, for an extension called a "pull request" extension, I was very surprised it couldn't *create* a pull request.  Seems like this would be fairly trivial compared to all the work done for the rest of this extension.

**Comment by adrienjoly** [comment]
Just found out that this extension provides the functionality: https://marketplace.visualstudio.com/items?itemName=jasonnutter.github-pr => I'm going to give it a try
